### PR TITLE
tests: use larger timeout with "timers can be stopped from the handler"

### PR DESCRIPTION
**Pull request by blueyed**
Seen on quickbuild:

    23:01:01,289 INFO  - not ok 1172 - timers can be stopped from the handler
    23:01:01,289 INFO  - # test/functional/eval/timer_spec.lua @ 154
    23:01:01,289 INFO  - # Failure message: test/functional/eval/timer_spec.lua:166: Expected objects to be the same.
    23:01:01,289 INFO  - # Passed in:
    23:01:01,289 INFO  - # (number) 3
    23:01:01,289 INFO  - # Expected:
    23:01:01,289 INFO  - # (number) 0
    23:01:01,289 INFO  - # stack traceback:
    23:01:01,289 INFO  - # 	test/functional/eval/timer_spec.lua:166: in function <test/functional/eval/timer_spec.lua:154>

Log: http://neovim-qb.szakmeister.net/build/24288
Ref: https://github.com/neovim/neovim/pull/10364

### CODE REVIEW

File: test/functional/eval/timer_spec.lua [hunk 1]
```diff
@@ -165,7 +165,7 @@ describe('timers', function()
     command("call timer_start(10, 'MyHandler', {'repeat': -1})")
     eq(0,eval("g:val"))
     run(nil, nil, nil, load_adjust(50))
-    retry(nil, 5000, function()
+    retry(nil, nil, function()
       eq(3, eval("g:val"))
     end)
```
Comment (justinmk) [inline test/functional/eval/timer_spec.lua]: what is the purpose of the run()? I think I should have removed it when I added the retry().

61e8adb25e6f37c3c9df

**Review by justinmk — APPROVED** [review]

**Comment by blueyed** [comment]
What's the quickbuild failure here?
(it says " Missing TAP Plan." - which log to look for then?)

**Comment by justinmk** [comment]
"Missing TAP Plan" is a side effect of the test runner not completing. Look for "terminated" or "not ok".

**Comment by blueyed** [comment]
Does it kill itself?
```
16:24:54,186 INFO  - ok 3431 - ui/mouse/input left drag changes visual selection
16:24:54,341 INFO  - Terminating launched command gracefully...
16:24:54,341 DEBUG - Executing command: sh /usr/home/quickbuild/buildagent/framework/configuration/org.eclipse.osgi/com/pmease/quickbuild/execution/kill-tree.sh 12872 
16:24:54,341 DEBUG - Command working directory: /usr/home/quickbuild/buildagent/bin
16:24:54,347 WARN  - Vim: Caught deadly signal 'SIGTERM'
16:24:54,360 WARN  - Terminated
16:24:54,367 WARN  - 
16:24:54,375 WARN  - Vim: Finished.
16:24:54,471 INFO  - Executing post-execute action...
```

**Pull request merged**